.github/workflows/api-swagger-ui.yml

```diff
@@ -0,0 +1,76 @@
+name: Build and Push Swagger-UI to R2 mainnet-api-docs.spacemesh.network
+
+on:
+  release:
+    types: [published]
+    
+jobs:
+  check-version:
+    runs-on: ubuntu-22.04
+    outputs:
+      go-sm-api-version: ${{ steps.go-sm-api-version.outputs.GO_SM_API_VERSION }}
+    steps:
+    - name: Checkout target repository on last release
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: 'refs/tags/${{ github.event.release.tag_name }}'
```

Comment:
Why does this need to be specified? Won't the event be triggered on the reference of the tag anyway?